var/spack/repos/builtin/packages/orfm/package.py

```diff
@@ -35,3 +35,5 @@ class Orfm(AutotoolsPackage):
     url      = "https://github.com/wwood/OrfM/releases/download/v0.7.1/orfm-0.7.1.tar.gz"
 
     version('0.7.1', 'fcf18283a028cea2af90663a76a73a2a')
+
+    depends_on('zlib', type='link')
```

Comment:
i think default (`build`+`link`) is what you need.

Reply:
Actually, `zlib` doesn't have any binaries in `prefix.bin`, so technically `type='link'` is more correct.